packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -3123,7 +3123,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
   @override
   double computeMaxIntrinsicWidth(double height) {
     _layoutText(maxWidth: double.infinity);
-    return _textPainter.maxIntrinsicWidth + cursorWidth;
+    return _textPainter.maxIntrinsicWidth + _caretMargin;
```

Comment:
Should we add this to `computeMinIntrinsicWidth` too?

Reply:
About this I am not very clear, @justinmc how to think about it?